tests/testing_utils.py

```diff
@@ -228,6 +228,18 @@ def process(sample):
                 "images": sample["image"],
             }
 
+    elif ds_name == "pile-val-backup":
+
+        def preprocess(example):
+            return {
+                "input_ids": processor.encode(example["text"].strip())[:max_seq_length]
+            }
+
+        ds = ds.map(preprocess, remove_columns=ds.column_names)
+        # Note: potentially swap filtering to pad for AWQ
+        ds = ds.filter(lambda example: len(example["input_ids"]) >= max_seq_length)
```

Comment:
Does this dataset need explicit preprocessing? Also why not truncate rather than filter?

Reply:
Just following practices that Brian has listed out - I think in general we should clean this up to be more consistent with the other datasets but ok for now.